go/cmd/run/main.go

```diff
@@ -1,20 +1,43 @@
 package run
 
 import (
+	"context"
+	"fmt"
+
 	"github.com/unkeyed/unkey/go/cmd/api"
 	"github.com/unkeyed/unkey/go/cmd/ctrl"
-	"github.com/urfave/cli/v3"
+	"github.com/unkeyed/unkey/go/pkg/cli"
 )
 
 var Cmd = &cli.Command{
 	Name:  "run",
 	Usage: "Run Unkey services",
 	Description: `Run various Unkey services including:
   - api: The main API server for validating and managing API keys
-  - ctrl: The control plane service for managing infrastructure`,
+  - ctrl: The control plane service for managing infrastructure
+
+EXAMPLES:
+    # Run the API server
+    unkey run api
+
+    # Run the control plane
+    unkey run ctrl
 
+    # Show available services
+    unkey run --help`,
 	Commands: []*cli.Command{
 		api.Cmd,
 		ctrl.Cmd,
 	},
+	Action: runAction,
+}
+
+func runAction(ctx context.Context, cmd *cli.Command) error {
```

Comment:
in an ideal world this would be autogenerated

like when a command has no action but subcommands, we generate such a help text

but that's really not urgent now, @coderabbitai can you create an issue for this and tag it as good first issue?

Reply:
We partially do that now but its not as descriptive as this one. But today i'll take another look when trying to figure out `mdx` generation using all description parts